packages/core/src/Table/stories/TableSection/TableEditable.tsx

```diff
@@ -595,6 +790,28 @@ const Table = <T extends Data>({
           }}
         />
       ) : undefined}
+      <HvDialog open={openViewDialog} onClose={() => setOpenViewDialog(false)}>
```

Comment:
This is starting to feel like "too much stuff" for a single sample. This is also not directly related to "editing".

If we can simplify the stuff above, we can keep it - otherwise I think we should move this "View" feature to another sample